hardhat.config.ts

```diff
@@ -58,7 +58,7 @@ const getCommonNetworkConfig = (networkName: eNetwork, networkId: number) => ({
 
 const mainnetFork = MAINNET_FORK
   ? {
-      blockNumber: 12290275,
+      blockNumber: 13410750, // Oct 13, 2021
```

Comment:
Lets pass here an env variable, so its possible to adjust its value depending on the task/test.
```
blockNumber: parseInt(process.env.FORKING_BLOCK)
```